Blockchain/Blockchain Bridge.md

```diff
@@ -0,0 +1,93 @@
+---
+tags: blockchain, blockchain-bridge
+author: Pham Ngoc Thanh
+---
+# Blockchain bridge
```

Comment:
Don't need to include the h1 because the title of the page will be automatically populated from the file's name.